lib/core/tools/text_tool/bounding_box.dart

```diff
@@ -0,0 +1,158 @@
+import 'dart:ui';
```

Comment:
No need for duplicate file. Original implementation from shapes tool PR can be move to `core\tools`